### PR TITLE
Fix issue #206 and issue #213 By making text bubbles step by step

### PR DESCRIPTION
**Pull request by Benjamin-Rosenfeld**

### CODE REVIEW

**Comment by polytroper** [comment]
Just a heads-up, I renamed TextBubble to Tip. Much shorter and to the point.

Now about your implementation. This technically works but uses some anti-patterns for our engine (and OOP more generally). Namely, it relies on DOM querying/manipulation for state management and communication between Tips instead of keeping that stuff confined to the Entity tree. You could say that your "model" is getting mixed with your "view".

Instead of querying the DOM for the next Tip, each one should be passed a `completeTip` callback from `Level` that activates the next Tip in the sequence. Does that make sense?

**Comment by Benjamin-Rosenfeld** [comment]
It shoule be better now, please check

**Comment by polytroper** [comment]
Your code looks mostly good. Let's actually first change some of the code you didn't write.

Let's change `Tip` so that that instead of destroying itself, it just sets an internal "completed" variable. Expose that variable on the `Tip` entity with a read-only property like `get completed() {return completed}`. Create a new method called `refreshDom` that updates the visibility and any other state of the dom node. Create another new method called `complete` that gets called where destroy is called now. `complete()` should:
* Set completed to true
* Call `refreshDom`
* Call `tipCompleted`
Then when `tipCompleted` is called, we loop over every `Tip` in our `Level` and call `refreshDom` on each of them.

Does that make sense?

**Comment by Benjamin-Rosenfeld** [comment]
Sorry, I forgot to push changes and tried to commit nothing.

**Comment by Benjamin-Rosenfeld** [comment]
uh oh, I accidently pushed the axis ticks. Can you please roll that back

File: Types/Entities/AxesTicks.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+function AxesTicks(spec) {
```
Comment (polytroper) [inline Types/Entities/AxesTicks.js:1]: Why is AxesTicks its own entity now? Should just be a part of Axes I think

**Review by polytroper — CHANGES_REQUESTED** [review]
Looks mostly good, asked for a few subtle changes. My big question about the Axes thing is that I don't see why a separate entity is needed

**Review by polytroper — APPROVED** [review]
Ok merging

**Pull request merged**